build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -89,6 +89,11 @@ spec:
                 maxItems: 5
                 minItems: 0
                 type: array
+              shortID:
```

Comment:
We should use a more explicit name like `logID`. Could you check how this is called in NSX-T?

Reply:
https://gitlab.eng.vmware.com/core-build/nsx/-/blob/nsx-keeper/mp/policy/policy-api/src/main/resources/api_spec/root/nsx-policy/policy-vpc.yml#L123, yea, I checked in NSX-T it is shortID, so use the same name.